src/pages/Accounts/Modals/SendFunds.tsx

```diff
@@ -130,7 +130,7 @@ export const AskTransferFundsModal: FC<AskSendFundsModalProps> = ({ isVisible, o
   }, [setAmount, getKusamaFee, recipientAddress]);
 
   const isConfirmDisabled = useMemo(() => (
-    !sender || !recipientAddress || Number(amount) <= 0 || Number(amount) > Number(formatKusamaBalance(sender?.balance?.KSM?.toString() || 0))
+    !sender || !recipientAddress || Number(amount) <= 0 || Number(amount) > kusamaBalanceToNumber(sender?.balance?.KSM?.toString() || 0)
```

Comment:
"sender.balance.KSM" should already be in correct format (all this comas are added by us)

May be u can use value from somewhere else, but original bigInt should be somewhere reachable

Reply:
fixed